robusta_krr/main.py

```diff
@@ -98,7 +98,7 @@ def run_strategy(
                     None,
                     "--resource",
                     "-r",
-                    help="List of resources to run on (Deployment, StatefulSet, DaemonSet, Job, Rollout). By default, will run on all resources. Case insensitive.",
+                    help="List of resources to run on (Deployment, StatefulSet, DaemonSet, Job, Rollout, StrimziPodSet). By default, will run on all resources. Case insensitive.",
```

Comment:
also add strimzi to :
_list_scannable_objects ...
            if kind in ("Rollout", "DeploymentConfig") and e.status in [400, 401, 403, 404]:

Reply:
done